@xen-orchestra/web/src/components/vdis/panel/cards/VdiInfosCard.vue

```diff
@@ -0,0 +1,120 @@
+<template>
+  <UiCard class="card-container">
+    <div class="vdi-label typo-body-bold">
+      <VtsIcon size="medium" name="fa:hard-drive" />
+      {{ vdi.name_label }}
+    </div>
+    <div class="content">
+      <VtsCardRowKeyValue>
+        <template #key>{{ t('uuid') }}</template>
+        <template #value>
+          {{ vdi.uuid }}
+        </template>
+        <template #addons>
+          <VtsCopyButton :value="vdi.uuid" />
+        </template>
+      </VtsCardRowKeyValue>
+      <VtsCardRowKeyValue>
+        <template #key>{{ t('description') }}</template>
+        <template #value>{{ vdi.name_description }}</template>
+        <template v-if="vdi.name_description" #addons>
+          <VtsCopyButton :value="vdi.name_description" />
+        </template>
+      </VtsCardRowKeyValue>
+      <VtsCardRowKeyValue>
+        <template #key>{{ t('tags') }}</template>
+        <template #value>
+          <UiTagsList v-if="vdi.tags.length > 0">
+            <UiTag v-for="(tag, index) in vdi.tags" :key="index" accent="info" variant="secondary">
+              {{ tag }}
+            </UiTag>
+          </UiTagsList>
+        </template>
+        <template v-if="vdi.tags.length > 0" #addons>
+          <VtsCopyButton :value="vdi.tags.join(', ')" />
+        </template>
+      </VtsCardRowKeyValue>
+      <VtsCardRowKeyValue>
+        <template #key>{{ t('status') }}</template>
+        <template #value>
+          <VtsStatus :status="vbdsStatus" />
+        </template>
+        <template v-if="vbdsStatus" #addons>
```

Comment:
No need to check `vbdStatus`, as it will always be defined:

```suggestion
        <template #addons>
```